shift_detector/checks/DistinctionCheck.py

```diff
@@ -9,8 +10,14 @@
 class DistinctionCheck(Check):
 
     def __init__(self, columns=[], num_epochs=10, relative_threshold=.1):
+        if columns and (not isinstance(columns, Iterable) or any(not isinstance(column, str) for column in columns)):
+            raise Exception("columns should be empty or a list of strings. Received: {}".format(columns))
         self.columns = columns
+
+        if not isinstance(num_epochs, int) or num_epochs < 1:
+            raise Exception("Num epochs should be an Integer greater than 0, but is {} instead:".format(num_epochs))
```

Comment:
Please split this in ValueError and TypeError as they are different causes for the exception